pages/client_login.tsx

```diff
@@ -83,6 +83,12 @@ const scopes = [
 
   // https://dev.twitch.tv/docs/api/reference#get-chatters
   "moderator:read:chatters", // for get chatters api (not currently used)
+
+  // https://dev.twitch.tv/docs/api/reference#get-shield-mode-status 
+  "moderator:read:shield_mode",  // for reading the channel's shield-mode status/settings (not currently used)
+  
+  // https://dev.twitch.tv/docs/api/reference#update-shield-mode-status 
+  "moderator:manage:shield_mode", // for managing the channel's shield-mode status/settings (not currently used)
```

Comment:
As noted in the docs, `moderator:manage:shield_mode` allows for both managing and reading of shield_mode status, because of this, we could get away with only using `manage` - similar to how we use `channel:manage:vips` in replacement of `channel:read:vips`
```suggestion

  // https://dev.twitch.tv/docs/api/reference#get-shield-mode-status  
  // https://dev.twitch.tv/docs/api/reference#update-shield-mode-status 
  "moderator:manage:shield_mode", // for managing the channel's shield-mode status/settings (not currently used)
```